Teatoller/giphyfetch
# #002 Project Setup

#### What does this PR do?
- Set up initial project files
#### Description of Task to be completed?
- [x] clear generic files
- [x] add react bootstrap
- [x] add css styles
- [x] add component functions
- [x] add App component
#### How should this be manually tested?
1. Create a folder on your machine `mkdir myproject`
2. `cd myproject`
3. Run `git clone https://github.com/Teatoller/giphyfetch.git`
4. cd `giphyfetch`
5. open **giphyfetch** in your chosen editor. For **vscode** enter `code .` on terminal and press enter.
6. `git checkout ft-project-setup-001`
7. Run `yarn install`
 
8. Run `yarn start`  the app in the development mode to open [http://localhost:3000](http://localhost:3000) to view it in the browser.
9. Check browser for display similar to  **screenshot: 1**
#### Any background context you want to provide?
-n/a
#### What are the relevant pivotal tracker stories?
- n/a
#### Screenshots (if appropriate)
**Screenshot: 1** Start up screen
![setup](https://user-images.githubusercontent.com/39020475/68953611-5001a380-07d3-11ea-9ef3-d233a88fb424.png)

#### Questions:
